stack.yaml

```diff
@@ -33,7 +33,7 @@ packages:
   extra-dep: true
 - location:
     git: https://github.com/input-output-hk/plutus-prototype
-    commit: e2e2711e6978002279b4d7c49cab1aff47a2fd43
+    commit: d2c7e65ab463de5515980af0778c5c9dcd79354d
```

Comment:
It's bad that commit from feature branch is used. It's also bad that master is not protected there, but it's out of scope of this PR.